libs/execution_context/ebpf_maps.c

```diff
@@ -29,6 +29,7 @@ typedef struct _ebpf_core_object_map
     ebpf_lock_t lock;
     ebpf_map_definition_in_memory_t inner_template_map_definition;
     bool is_program_type_set;
+    bool supports_context_header;
```

Comment:
Why are these on the base object and not the prog_array one?

Reply:
Actually, scratch that. This is only being set for prog_array, hash_of_map, and array_of_map, so that's probably fine.